AcceptanceTest.md

```diff
@@ -0,0 +1,23 @@
+# AcceptanceTest for QualityToolsExtension.jar
+
+Due to the difficulty of JUnit tests developers created the following acceptance test to verify the start up functionality of the BlueJ application. 
+
+
+# Steps 
+
+1) Gradle build the project with JDK version 11
```

Comment:
As I progress through this document, I begin to understand that this is intended more as a "one off" acceptance criterion than something useful for longer duration of product development and maintenance.

If this is the case, then it should appear as "Acceptance Criteria" in the associated issue, rather than as a document in the repository.

Consider copying the verification steps to the appropriate issue(s) and deleting this document.

Reply:
After discussing with the group, having a JTextField displayed in the quality tools extension tab serves no purpose. The intent was to match the #112 user story "picture display"